lib/arjdbc/postgresql/connection_methods.rb

```diff
@@ -30,6 +30,7 @@ def postgresql_connection(config)
     config[:username] ||= ( config[:user] || ENV['PGUSER'] || ENV_JAVA['user.name'] )
     config[:password] ||= ENV['PGPASSWORD'] unless config.key?(:password)
     properties = ( config[:properties] ||= {} )
+    properties['stringtype'] = 'unspecified' # for simple strings looking like UUIDs
```

Comment:
would be nice to respect user setting if there's any (as we allow users to specify `properties`) ...
```suggestion
    properties['stringtype'] ||= 'unspecified' # for simple strings looking like UUIDs
```

Reply:
I think that ideally there can be a separate test suite for different `stringtype` values. Possible values are `VARCHAR` (the default) and the only other option is `unspecified` IIRC.

However, setting `stringtype = VARCHAR` (as it was before) will break queries if variables look like UUIDs.

So, I think that the only correct option here is `unspecified`. However, I'll let you to decide. If you say that this user setting should be respected then ok.